syntax = "proto2";

package protoAuth;

message LoginRespose
{
	required int32 reqId = 1;
	required eLoginResult result = 2;

   	enum eLoginResult{
    		SUCCESS = 0;
    		INVALID_PASSWORD = 1;
    		EMAIL_NOT_FOUND = 2;
    		INTERNAL_SERVER_ERROR = 3;
  	}
	
}
